labshare-client/src/components/CheckboxGroup.vue

```diff
@@ -1,40 +1,36 @@
 <i18n>
 {
   "en": {
-    "bsl1": "Work under BSL2 regulations/safety standards",
+    "bsl1": "Work under BSL1 regulations/safety standards",
     "bsl2": "Work under BSL2 regulations/safety standards",
-    "bsl3": "ork under BSL3 regulations/safety standards",
+    "bsl3": "Work under BSL3 regulations/safety standards",
     "bsl4": "Work under BSL4 regulations/safety standards",
     "qpcr": "qPCR",
-    "rnaExperience_isolation": "RNA isolation (with kit)",
-    "sample_processing": "(Pre)Processing of clinical samples",
+    "rnaExperience_isolation": "Work with RNA (esp. isolation with kit)",
+    "sample_processing": "(Pre-)Processing of clinical samples",
     "elisa": "ELISA",
-    "primerProduction": "Primerherstellung",
-    "dataAnalysis": "Datenanalyse",
+    "primerProduction": "Primer production",
+    "dataAnalysis": "Data analysis",
 
-    "calibratedPipetteSet": "Calibrated Pipette Set",
+    "calibratedPipetteSet": "Calibrated pipette set",
     "rtThermocycler": "Thermocycler RT",
     "qpcrThermocycler": "Thermocycler qPCR",
-    "primerProduction": "Primer Production",
-    "reverseTransKit": "Reverse Transcriptase Kit",
-    "pcrMasterMix": "PCR Master Mix",
-    "rnaExtractionDevice": "RNA Extraction Device",
-    "rnaExtractionKit": "RNA Extraction Kit",
+    "reverseTransKit": "Reverse transcriptase kit",
+    "pcrMasterMix": "PCR master mix",
+    "rnaExtractionDevice": "RNA extraction device",
+    "rnaExtractionKit": "RNA extraction kit",
 
-    "virology": "Virologie",
-    "rnaExperience_isolation": "Arbeit mit RNA (insbes. Isolation)",
-    "qPCR": "Allgemein qPCR",
-    "elisa": "ELISA",
-    "protocolProduction": "Protokollerstellung",
+    "virology": "Virology",
+    "protocolProduction": "Writing of protocols",
 
-    "jobTraining": "Berufsausbildung",
+    "jobTraining": "Job training",
     "mtla": "MTLA",
     "bta_cta": "BTA/CTA",
-    "bachelor": "Bachelor",
-    "master": "Master",
-    "doctorate": "Promotion",
+    "bachelor": "Bachelor (completed)",
+    "master": "Master (completed)",
+    "doctorate": "PhD-Student",
```

Comment:
The hyphen is unnecessary, and all of these should probably be put into title case (on this page, meaning each word gets an initial capital letter)